.github/CODEOWNERS

```diff
@@ -1,3 +1,3 @@
 # See https://help.github.com/articles/about-code-owners/
 
-* @kentico-michaelb @Kentico/developer-relations
+* @kontent-ai-michaelb @Kontent-ai/developer-relations
```

Comment:
I don't think @kontent-ai-michaelb is an existing user name. 😄 I think we can change codeowners to just devrel.